types/recharts/index.d.ts

```diff
@@ -580,7 +581,7 @@ export interface ScatterProps {
 	points: any[];
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?